release/index.d.ts

```diff
@@ -0,0 +1,9 @@
+import { DataType } from "./src/datatype";
```

Comment:
I think it is should be fine. I wish typescript was more consistent in this behavior.